From 06923af404393a5dbde22a0e5e62ddea1b7cd558 Mon Sep 17 00:00:00 2001
From: "H.J. Lu" <hjl.tools@gmail.com>
Date: Fri, 30 Aug 2019 08:34:26 -0700
Subject: [PATCH 1/5] gas: Add md_cons_worker

Add md_cons_worker to allow backend to track hard-coded opcodes in
instruction stream.

	* read.c (cons_worker): Call md_cons_worker if it is define.
---
 gas/read.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/gas/read.c b/gas/read.c
index 58d79b2787..d4bb0338d9 100644
--- a/gas/read.c
+++ b/gas/read.c
@@ -3977,6 +3977,10 @@ cons_worker (int nbytes,	/* 1=.byte, 2=.word, 4=.long.  */
   md_cons_align (nbytes);
 #endif
 
+#ifdef md_cons_worker
+  md_cons_worker (nbytes);
+#endif
+
   c = 0;
   do
     {
-- 
2.21.0

